partnercenter/azext_partnercenter/models/plan.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from msrest.serialization import Model
+
+
+class Plan(Model):
```

Comment:
By inheriting from Model, when our model class instance is returned by command methods in custom.py, it will automatically get converted to dict by the CLI framework.